package/src/library/DatePicker/index.tsx

```diff
@@ -58,24 +58,22 @@ const DatePicker = ({
   }
 
   return (
-    <div className="relative">
-      <Popover.Root>
-        <Popover.Trigger>
-          <Input.Root className="mb-2">
-            <Input.Input
-              type="text"
-              placeholder={placeholder}
-              value={getInputValue(rest.selected)}
-              readOnly
-            />
-            <Input.Icon icon={icon} />
-          </Input.Root>
-        </Popover.Trigger>
-        <Popover.Content>
-          <Calendar {...rest} className="shadow-lg" />
-        </Popover.Content>
-      </Popover.Root>
-    </div>
+    <Popover.Root>
```

Comment:
Joga isso dentro do App e testa de várias formas, tu tá só jogando a alteração aí dentro do DatePicker e confiando que vai funcionar em todos os casos. Usa o App de várias formas, é melhor usar o App.tsx que temos aí no Visu pra testar do que deixar pra testar só lá no projeto e descobrir que deu errado. Dá uma olhada na lib do radix também sobre o [Popover](https://www.radix-ui.com/primitives/docs/components/popover) e as variáveis de tamanho (pesquisa por CSS variables lá na página do Popover) que ele trás etc... Eu não sei se oq tu fez aí vai funcionar pq eu não rodo as prs aqui na minha máquina pra testar, mas eu sei que tu não alterou o App e não testou essa alteração

Reply:
Ao utilizar o componente `DatePicker` (no App por exemplo) eu não tenho como mudar a estilização do que está dentro dentro dele.

Mesmo que eu jogue um `w-full` nele, ele não repassa para o popover, o que limita a largura dele.

O nosso popover não possui css para largura no trigger, apenas no cotent.

Assim, para que no uso do `DatePicker`, ele fique com a largura full dentro de onde for utilizado, se faz necessário alterar o trigger para `w-full`.

Fazendo isso eu consigo, então, passar para o `DatePicker`, no seu uso, larguras diferentes que vão fazer o trigger se adaptar ao full, como um container.

Assim posso passar um 
```
<DatePicker
          className="w-1/2"
          ...
```

=======

Se ficar como estava
```
<div className="w-full">
      <Popover.Root>
        <Popover.Trigger>
```

Não tenho o controle da largura no DatePicker

Reply:
No Popover ele não tem props nativas no Radix para width, sua largura se adapta pelos filhos, no caso o calendar que fica no content.

Se eu quiser ampliar a largura do popovr eu enho que jogar uma className com w.

Como a ideia é que o trigger se adapte, e não o content, por isso joguei ali.

Agora, posso criar uma props de width para o DatePiker para passar no uso, que regularia o trigger do popover.

====

```
<Popover.Root>
      <Popover.Trigger className={width}>
      ...
```

```
<DatePicker
          width="w-full"
          placeholder="Selecione uma data"
          icon={<CalendarIcon />}
          mode="multiple"
          options={{
            weekday: 'long',
            year: 'numeric',
            month: 'long',
            day: 'numeric',
          }}
        />
```